diff --git a/android/build.gradle b/android/build.gradle
index b2803c4f830c53c52831695190c30e5f0a803ddd..ea1069292aa9280cecd9b97fd81f925d6e6c9433 100644
--- a/android/build.gradle
+++ b/android/build.gradle
@@ -51,7 +51,7 @@ android {
     buildTypes {
         release {
             minifyEnabled false
-            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
         }
     }
 }
